logging/interceptor.go

```diff
@@ -172,7 +228,7 @@ func addAccountIDField(ctx context.Context, fields logrus.Fields) (context.Conte
 
 	fields[DefaultAccountIDKey] = accountID
 
-	return metadata.AppendToOutgoingContext(ctx, DefaultAccountIDKey, accountID), err
+	return ctx, err
```

Comment:
If we are not changing anything in context, can we remove it from return statement ?